### PR TITLE
8882 comment fixes

### PR DESCRIPTION
**Pull request by sttts**
Fixes mesosphere/kubernetes-mesos#330

### CODE REVIEW

File: contrib/mesos/pkg/executor/executor.go [hunk 1]
```diff
@@ -243,6 +243,12 @@ func (k *KubernetesExecutor) Disconnected(driver bindings.ExecutorDriver) {
 }
 
 // LaunchTask is called when the executor receives a request to launch a task.
+//
```
Comment (jdef) [inline contrib/mesos/pkg/executor/executor.go]: why the blank line?

File: contrib/mesos/pkg/executor/executor.go [hunk 2]
```diff
@@ -467,7 +473,7 @@ func (k *KubernetesExecutor) launchTask(driver bindings.ExecutorDriver, taskId s
 	task.podName = podFullName
 	k.pods[podFullName] = pod
 
-	// Send the pod updates to the channel.
+	// Send the pod update to the kubelet via the pod config update channel.
```
Comment (jdef) [inline contrib/mesos/pkg/executor/executor.go]: `// send the latest snapshot of the set of pods to the kubelet via the pod update channel`

**Comment by jdef** [comment]
couple of minor nits, otherwise LGTM

**Comment by sttts** [comment]
Addressed both nits.

**Pull request merged**